### PR TITLE
consegna primi 20 script - Fase2

### PR DESCRIPTION
**Pull request by fmassouNTTDATA**

### CODE REVIEW

File: src/main/java/it/pn/frontend/e2e/section/mittente/DettaglioNotificaMittenteSection.java [hunk 1]
```diff
@@ -257,4 +226,15 @@ public void clickLinkAttestazioneOpponibileAvvenutoAccesso() {
             Assert.fail("Non riuscito ad trovare il link con errore: "+e.getMessage());
         }
     }
+
+    public void verificaInviPECInCorso() {
```
Comment (leleOFA) [inline src/main/java/it/pn/frontend/e2e/section/mittente/DettaglioNotificaMittenteSection.java]: change to verificaInvioPECInCorso

File: src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java [hunk 2]
```diff
@@ -226,8 +226,8 @@ public void clickMenuButtonBlocca() {
     public boolean siVisualizzaApiKeyConTesto() {
         By apiKeyBy = By.xpath("//td[div/div[contains(@class,'MuiBox-root css-4l7hgf')]]");
         List<WebElement> apiKeyList = this.elements(apiKeyBy);
-        for (int i = 0; i < apiKeyList.size(); i++) {
-            if (apiKeyList.get(i).getText() == null){
+        for (WebElement webElement : apiKeyList) {
+            if (webElement.getText() == null) {
```
Comment (leleOFA) [inline src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java]: insert stronger control

File: src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java [hunk 3]
```diff
@@ -247,8 +247,8 @@ public boolean siVisualizzaNomeEDataConTesto() {
     public boolean siVisualizzaGruppoConTesto() {
         By gruppoCellBy = By.xpath("//td[div/div/div/div/div/span[contains(@class,'css-t63gu0')]]");
         List<WebElement> gruppoCellList = this.elements(gruppoCellBy);
-        for (int i = 0; i < gruppoCellList.size(); i++){
-            if (gruppoCellList.get(i).getText() == null){
+        for (WebElement webElement : gruppoCellList) {
+            if (webElement.getText() == null) {
```
Comment (leleOFA) [inline src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java]: as before

File: src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java [hunk 4]
```diff
@@ -258,8 +258,8 @@ public boolean siVisualizzaGruppoConTesto() {
     public boolean siVisualizzaStatoConTesto() {
         By statoCellBy = By.xpath("//td[div/div/div/div[@role='button' ]]");
         List<WebElement> statoCells = this.elements(statoCellBy);
-        for (int i = 0; i < statoCells.size(); i++) {
-            if (statoCells.get(i).getText() == null) {
+        for (WebElement statoCell : statoCells) {
+            if (statoCell.getText() == null) {
```
Comment (leleOFA) [inline src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java]: as before

File: src/main/java/it/pn/frontend/e2e/section/mittente/DettaglioNotificaMittenteSection.java (hunk 1 above)
Comment (delfinoroNTTDATA) [reply src/main/java/it/pn/frontend/e2e/section/mittente/DettaglioNotificaMittenteSection.java]: done

File: src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java (hunk 2 above)
Comment (delfinoroNTTDATA) [reply src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java]: done

File: src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java (hunk 3 above)
Comment (delfinoroNTTDATA) [reply src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java]: done

File: src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java (hunk 4 above)
Comment (delfinoroNTTDATA) [reply src/main/java/it/pn/frontend/e2e/pages/mittente/ApiKeyPAPage.java]: done

**Review by leleOFA — APPROVED** [review]

**Pull request merged**